Docker/test/selenium/specs/repo-client/client-item.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const Util = require( 'wdio-mediawiki/Util' );
+const assert = require( 'assert' );
+const WikibaseApi = require( 'wdio-wikibase/wikibase.api' );
+
+describe( 'Client Item', function () {
+
+	it( 'Special:NewItem should not be accessible on client', function () {
+
+		browser.url( process.env.MW_CLIENT_SERVER + '/wiki/Special:NewItem' );
+		$( 'h1#firstHeading' ).waitForDisplayed();
+		const notFoundText = $( 'h1#firstHeading' ).getText();
+		assert( notFoundText === 'No such special page' );
```

Comment:
Do we really want to bind to the specific text here, or would it be better to use the message name (`?uselang=qqx`) or maybe just the http code (404)?

Reply:
yup sounds good